docs/guides/Screen-Nav-Options.md

```diff
@@ -19,7 +19,8 @@ class MyScreen extends React.Component {
 
 Or, each option can be a function that takes the following arguments, and returns the value of the option.
 
-- `navigation` - the [navigation prop](/docs/intro/navigation-prop) for the screen, with the screen's route at `navigation.state`
+- `navigation` - The [navigation prop](/docs/intro/navigation-prop) for the screen, with the screen's route at `navigation.state`
+- `viewOptions` - Options specific to the view, e.g. - header, tabBar or drawer
```

Comment:
@ericvicenti we have to finish write documentation about screen nav options for drawer and tab bar before I add that there. I'll create an issue for it.